appeals/api/Dockerfile

```diff
@@ -4,20 +4,18 @@ COPY package*.json ./
 
 COPY packages ./packages
 
-COPY appeals/api apps/api
+COPY appeals/api appeals/api
 ENV NODE_ENV=production
 RUN npm ci
 COPY appeals/api/.env.development ./.env
-COPY appeals/api/init.sh ./init.sh
 
-ENV SWAGGER_JSON_DIR=./apps/api/src/server/openapi.json
+# relative to cwd
+ENV SWAGGER_JSON_DIR=./src/server/openapi.json
 ENV DATABASE_URL=$DATABASE_URL
 
-WORKDIR /src/app/apps/api
+WORKDIR /src/app/appeals/api
```

Comment:
Hey @beejjacobs I had left the internal Docker FS layout the same to minimise the changes in scripts and reuse the startup scripts. Not so sure what's the right answer... This is definitely clearer, if it works!

Reply:
It was the directory causing the issue. The lock file saves packages are required by each package, and that is referenced by path. So it was installing those that matched apps/api instead of appeals/api